packages/remix-vercel/package.json

```diff
@@ -16,12 +16,13 @@
   },
   "devDependencies": {
     "@types/supertest": "^2.0.10",
-    "@vercel/node": "^1.8.3",
+    "@vercel/node": "^2.0.1",
```

Comment:
```suggestion
    "@vercel/node": "^2.2.0",
```

Reply:
@mcansh This means we'll have to re-generate `yarn.lock`

Reply:
👍 i was AFK when i got the earlier notification, doing that now